README.md

```diff
@@ -11,8 +11,29 @@ A small ffmpeg Docker image. Contains only `ffmpeg`, not `ffserver`, `ffprobe` a
 docker pull aviaryan/ffmpeg
 ```
 
-| | |
-|-------------|--------------|
-| Download    | 20 MB |
-| Downloaded  | 50 MB |
-| Docker Hub  | https://hub.docker.com/r/aviaryan/ffmpeg/ |
+<table>
+    <tr>
+        <td>
+            Download
+        </td>
```

Comment:
@singhpratyush Thanks. Please convert it to single line style i.e. like 

```
<td>Download</td>
```

Cheers.